package.json

```diff
@@ -6,9 +6,10 @@
     "start": "node ./bin/www",
     "test": "export NODE_ENV=test && mocha spec"
   },
-  "engines" : { "node" : ">=4.1.0 <5.5.0" },
+  "engines": {
+    "node": ">=4.1.0 <5.5.0"
+  },
   "dependencies": {
-    "bluebird": "^3.1.1",
```

Comment:
Thanks for removing this dependency and using the built-in promises. Removing external unnecessary dependencies is always a good thing. 😉